src/commons/Reporter.ts

```diff
@@ -138,8 +138,12 @@ export namespace Reporter {
     if (stepStatus !== undefined) {
       status = stepStatus;
     }
+    const stepContent: Object = {
+      content: command.body,
+      name: command.bodyLabel,
+    };
     // tslint:disable-next-line:no-unsafe-any
-    createStep(command.title, command.body, command.bodyLabel, status);
+    allureReporter.addStep(command.title, stepContent, status);
```

Comment:
`stepContent` is an object. Is it logged nicely by the reporter? Usually in Javascript this looks like `[Object object]` in the log

Reply:
Felix Code please consult

Reply:
allures api change to : 
`addStep(title, [{content, name = 'attachment'}], [status]) - add step to test.
title (String) - name of the step.
content (String, optional) - step attachment
name (String, optional) - step attachment name, attachment by default.
status (String, optional) - step status, passed by default. Must be "failed", "passed" or "broken"`

that's the reason for object now